### ISSUE TITLE
Remote Development Icon going red and not showing any error I could use to figure out what is wrong

### ISSUE THREAD

**Issue by Toasterson** [lapce/lapce#1109]
## Lapce Version
0.2.0
## System information
I am trying to connect from a macOS 12.5.1 (21G83) Early 2015 Air to a server running illumos current and another one running ArchLinux current amd64
## Describe the bug
I try to connect to an SSH host and type in the name as I have it configured in my ~/.ssh/config `oibuild`, Lapce tries to connect but fails and then show a red connection icon. 
For ssh I use the command `ssh oibuild` and this config.
```ssh_config
Host oibuild
  User *****
  Hostname build
  ProxyJump ****@$PUBLIC_IP
  ForwardAgent yes
  AddressFamily inet
  TCPKeepAlive=yes
  ServerAliveCountMax=20
  ServerAliveInterval=60
```
I assume something with the ssh command fails but I can't figure out what as I can't find the log and no documentation on how to create one. I am a rust developer and illumos Contributor so if I need to compile pace from source I am happy to do that to get logs but I would like some mentoring to fix this bug at least.

**Comment by vldptn178187** [comment]
Not official answer: Run Lapce from command line like that /Applications/Lapce.app/Contents/MacOS/lapce (Mac OS X)
then you can follow app logs including ssh logs

**Comment by Toasterson** [comment]
Unfortunately not with the version I downloaded from the site. Do I need a debug build?

**Comment by virtualritz** [comment]
TLDR; Docs on remote development SSH format suck. I tried `user@host:port`  -- nothing. No error, only the icon turns red. Also tried a full command `ssh user@host -p port`, same. No output on the terminal from which I launched `lapce` either. I'll try with a debug build next.
 
Installed `lapce` after six months again to give it another spin. Couldn't get this to work. This is basic for me. Back to VSCode. :(

If someone tells me what I am doing wrong I'm happy to improve the **Remote Development** docs section and submit a PR.

**Comment by virtualritz** [comment]
See also #906.
